test/scoping.js

```diff
@@ -8,10 +8,11 @@ describe('scoping', function () {
     };
 
     function Module () {
-      this.foo = 'bar';
+      return this.foo = 'bar';
     }
 
     Module.prototype.method = function() {
+      Module()
```

Comment:
This is not quite right, go and Google "javascript scoping" and update

Reply:
I've updated this now.

Cheers
Laura